site/content/en/docs/tasks/troubleshooting/troubleshooting_provreq.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Troubleshooting Provisioning Request in Kueue"
+date: 2024-05-20
+weight: 3
+description: >
+  Troubleshooting the status of a Provisioning Request in Kueue
+---
+
+This document helps you troubleshoot ProvisioningRequests, an API defined by [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/blob/4872bddce2bcc5b4a5f6a3d569111c11b8a2baf4/cluster-autoscaler/provisioningrequest/apis/autoscaling.x-k8s.io/v1beta1/types.go#L41).
+
+Kueue creates ProvisioningRequests via the [Provisioning Admission Check Controller](/docs/admission-check-controllers/provisioning/), and treats them like an [Admission Check](/docs/concepts/admission_check/). In order for Kueue to admit a Workload, the ProvisioningRequest created for it needs to succeed.
+
+## Before you begin
+
+Before you begin troubleshooting make sure your cluster meets the following requirements:
+- Check your cloud provider's documentation to determine the minimum versions that support ProvisioningRequest. If you use GKE, your cluster's version should be at least `1.28.3-gke.1098000` with the Cluster Autoscaler at least `28.122.0`
+- Kueue's version is at least `0.5.3`
+- You have enabled the `ProvisioningACC` in [the feature gates configuration](/docs/installation/#change-the-feature-gates-configuration)
+- You use a type of nodes that support ProvisioningRequest. It may vary depending on your cloud provider.
```

Comment:
Let's put it as second point, since it's related to `cluster supports ProvisioningRequest`

Reply:
Done